backend/src/app/entrypoints/api/routes/copilot_api.py

```
@@ -0,0 +1,153 @@
+        # Log the exception details for server-side debugging.
+        # logger.error(f"Exception during Copilot action: {e}", exc_info=True)
+        raise HTTPException(status_code=500, detail=f"An unexpected error occurred: {str(e)}")
+
+
+@router.post("/mock", response_model=CopilotMockResponse)
+async def mock_copilot_endpoint(request_data: CopilotMockRequest):
+    """
+    Mock endpoint for Storybook development and testing.
+    Returns predictable responses based on the 'scenario' parameter.
+    """
+async def mock_copilot_endpoint(request_data: CopilotMockRequest = Body(...)):
```

Ruff (0.11.9)

Redefinition of unused `mock_copilot_endpoint` from line 103

(F811)